frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -597,6 +638,10 @@ export type CreateSubscriptionYAMLProps = {
   location: Location;
 };
 
+export type PackageManifestPageProps = {
+  obj: CatalogSourceKind;
+} & ListPageProps;
+
```

Comment:
This type def should be named `<<ComponentName>>Props` (e.g. `OperatorsProps`). I suggested renaming this component, so adjust accordingly. `PackageManifestPageProps` collides with an existing type definition in `./packages/operator-lifecycle-manager/src/components/package-manifest.tsx`.